account_invoice_inter_company/tests/test_inter_company_invoice.py

```diff
@@ -340,6 +340,7 @@ def setUpClass(cls):
         if "company_ids" in cls.env["product.template"]._fields:
             # We have to do that because the default method added a company
             cls.product_consultant_multi_company.company_ids = False
+            cls.product_consultant_multi_company.product_tmpl_id.company_ids = False
```

Comment:
This is using a field that doesn't exist without the other module installed, so this highlights a problem in this PR that should be solved other way.

Reply:
The same goes for the line above it. The existing comment already reads:
```
# if product_multi_company is installed
```
The dependency is checked with the existing conditional. I don't see the issue here, the test is just very thorough.

Reply:
OK, I see... yes, it's not ideal, but let's continue for not blocking this.

Reply:
Any system that cannot by its very nature be orthogonal is not ideal, hence ERP's are not ideal. Sure beats spreadsheets though :stuck_out_tongue_winking_eye: 
As always, thanks for going into the fine details @pedrobaeza!

Reply:
Thanks to you for solving this problem.